### DISCUSSION TITLE
Adding docs

### DISCUSSION THREAD

**Discussion by kbulatovic** [btd/rollup-plugin-visualizer#161] [Ideas]
Hey. I wanted to hop onboard to fix the bug with `include`, `exclude` stuff, and i just found it hard to develop with no proper gui. I used to open up bundled html files, then edit some source there, and check the changes, but thats lacklustre experience.

Anyway, what do you think about adding `/docs` dir where we could create a minimal representation of templates available using vite+preact+router for examples with currently some hardcoded data, then host it as a live demo on github pages, and at the same time have a breeze of local dev?

**Comment by btd** [comment]
Hi @kbulatovic, 

It is interesting idea about docs. It makes sense to me, but i strictly against any custom development. IMO if it is possible to make docs using static site generators/MDX or what is modern now - makes much more sense to me.

From your description i have a feeling you are mixing something like storybook and actual docs explaining what and how it is working. I do not think storybook is any how useful there. It makes more sense to use vite preview option or something very similar.
If we need to make changes to make it work lets do this.

**Reply by kbulatovic** [reply]
No, storybook is an overkill nothing like that. And installing some popular SSG tools is a big lift for simple previews. I wasn't really gonna include that much writing at all, just a simple dev server for preview existing templates. So the changes have like 10 files, with Vite dev server using `import.meta.glob` to generate routes to template examples, and its really simple to link it to github pages for a live demo, for people who wanna see how templates look and feel. Il scrap up something on the weekend, and open a PR.
